docs/tutorials/index.md

```diff
@@ -31,12 +31,43 @@ Note that it is just a rough classification, and that there are several minor de
 ### Planning simulation
 
 Planning simulation is a simulation type that uses simple dummy data and tests mainly planning/control components.
+It brings the car to the destination while avoiding pedestrians and surrounding cars.
+
+#### How the planning simulation is achieved
+
+- Generates the path to the destination
+- Controls the car along the generated path
+- Detect humans and surrounding cars for the safe operation
+
+In the real-world operation, we need to detect traffic signs and signals to follow the rules, but in this simulation, we only do path generation, path following, and obstacle avoidance.
 
 ### Rosbag replay simulation
 
-Rosbag replay simulation is a simulation type that uses Rosbag data and tests mainly perception modules.
+Rosbag replay simulation is a simulation type that uses Rosbag data and tests mainly perception modules, such as localization.
 Sometimes it is used for endurance tests by repeatedly playing back the data.
 
+#### Localization
+
+Rosbag simulation performs localization, which is the process of figuring out the vehicle pose on the map or on a reference coordinate system.
+In other words, localization is the process of determining the vehicle pose on a local area map (e.g., map of the Tokyo area) or on the earth, or on a specific coordinate system (e.g., the tunnel entrance or the initial pose).
+Usually, the word "localization" means estimating the position of an object, but in autonomous driving, this word also includes estimating the pose of a vehicle.
```

Comment:
I feel this section should be explained on other pages like design documents.

Reply:
Agreed. I do not think that we should start defining how modules/components work in a tutorial about simulators.

Reply:
Where should it be placed? I thought `/design` can be one candidate

Reply:
Maybe the page that will be added by #86?

Reply:
How is the update on this?

Reply:
Please confirm that in #86.